packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -0,0 +1,133 @@
+2. Create a union with all the pieces of your icon. This will merge individual pieces 
+together to create one shape. You can find the Union tool in Sketch's toolbar at the 
+top or go to “Layer > Combine > Union".
+3. Apply the layer style “icon/color-text-icon" from one of the Paste themes to the 
+combined shape. In the layer list, you should only have this shape and the hidden 
+“source" group.
+4. Rename the combined shape to “[icon name]".
+5. Make sure the icon is centered on the artboard both vertically and horizontally. 
+If you’re working with an asymmetric shape, you might need to adjust it manually to make 
+sure it’s [optically centered](https://blog.marvelapp.com/optical-adjustment-logic-vs-designers/). 
+For example, you might need to move a “play" icon a bit 
+right-of-center to account for the low visual weight on its right side.
+6. Make sure there are no transforms on the shape. If there are, go to “Layer > Combine > Flatten".
+![Number of transforms preview](../../assets/images/icon-guide/formatting-an-icon-1.png)
+7. This is what your layer and sidebar should look like:
+![Example of what Sketch should look like](../../assets/images/icon-guide/formatting-an-icon-2.png)
+8. If you’re working on a Paste icon, place the “24/[icon name]" symbol on the 
+"20/[icon name]" and "16/[icon name]" artboards. Resize them to 20px × 20px and 
+16px × 16px respectively. Fix the size of the layer horizontally and vertically.
+![Example of how to size icons](../../assets/images/icon-guide/formatting-an-icon-3.png)
+9. Rename the layer to “icon".
+
+
+## Exporting and adding the icon
+
+1. Make sure you have the [SVGO compressor Sketch plugin](https://www.sketch.com/extensions/plugins/svgo-compressor/) installed.
+2. Export the 24px icon artboard (not the combined shape) as an SVG. The template artboard 
+is already set up to do this.
+3. Make sure the SVG code is clean.
+    1. You shouldn’t see `id`. If you do, make sure the SVGO compressor plugin is properly installed.
+    2. You shouldn’t see `transform`. If you do, flatten your shape.
+    3. You should see `width=”24" height="24"`. If you don’t, check that the artboard you’re 
+    exporting is 24px × 24px and you’re exporting the artboard itself.
+4. Commit your changes in Abstract and request a review from the Paste design team.
+5. Add the SVG to your code.
+    1. If the icon will be added to Paste, attach the SVG file to the original Github issue. 
+    The Paste team will add it to the repo and let you know when it’s done.
+    2. If it won’t be added to Paste, use the [svg-to-react](https://github.com/twilio-labs/svg-to-react) 
+    utility to create an accessible and extendable React component.
+
+
+## Troubleshooting
+
+If you run into any issues with creating your icon, double check you've followed 
+all the steps. In the past, we've seen icons that had small artifacts or missing 
+pieces because of the order of the paths within the union.
```

Comment:
Change to "In the past, we've seen icons that had small artifacts or missing pieces because of the order of layers within a combined shape or union. Try moving layers around to get the shape you need."